webapp/components/Notification/Notification.vue

```diff
@@ -1,40 +1,90 @@
 <template>
-  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
+  <ds-space :class="{ read: notificationData.read, notification: true }" margin-bottom="x-small">
     <client-only>
       <ds-space margin-bottom="x-small">
-        <user-teaser :user="from.author" :date-time="from.createdAt" />
+        <user-teaser :user="notificationData.triggerer" :date-time="notificationData.createdAt" />
```

Comment:
`triggerer` is not actually a word...
maybe `submitter`?

Reply:
I have found the word `triggerer` in DeepL.
And `submitter` will not hit the meaning good enough in my eyes.